Ginger/GingerCore/Variables/VariableBase.cs

```diff
@@ -345,6 +345,22 @@ public static void GetListOfUsedVariables(object item, ref List<string> usedVari
                                                 usedVariables.Add(value.ToString());
                                     }
                                 }
+                                else if(mi.Name == "ValueCalculated" && mi.DeclaringType.Name == "FlowControl")
```

Comment:
Add a comment explaining that this condition is added for Set variable flow control type